deeds-dapp-webapp/src/main/webapp/vue-app/js/Constants.js

```diff
@@ -0,0 +1,4 @@
+export const SLP_address = '0x960bd61d0b960b107ff5309a2dcced4705567070';
```

Comment:
why not using VueX global variables, where those addresses are already declared ?

Reply:
see https://github.com/Meeds-io/deeds-dapp/blob/3c4a5bbb1f606cf1d666578e0c611c6197e71f48/deeds-dapp-webapp/src/main/webapp/vue-app/main.js#L264